### PR TITLE
created catalog with grid

### PR DESCRIPTION
**Pull request by maksym-mishchanchuk**
[DEMO LINK](https://maksym-mishchanchuk.github.io/layout_catalog_grid/)

### CODE REVIEW

File: src/styles/blocks/_catalog.scss [hunk 1]
```diff
@@ -0,0 +1,30 @@
+@import "src/styles/utils/_variables.scss";
+@import "src/styles/utils/_mixins.scss";
+
+.catalog {
+  display: grid;
+  //grid-template-columns: repeat(1, $card-width);
```
Comment (vsuslov29) [inline src/styles/blocks/_catalog.scss:6]: коммент

**Review by vsuslov29 — APPROVED** [review]
WELL DONE!

File: src/styles/blocks/_catalog.scss [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+    grid-template-columns: repeat(2, $card-width);
+  }
+
+  @media (min-width: 768px) {
+    grid-template-columns: repeat(3, $card-width);
+  }
+
+  @media (min-width: 1024px) {
+    grid-template-columns: repeat(4, $card-width);
+  }
```
Comment (Anna-Tkachenko) [inline src/styles/blocks/_catalog.scss:19]: Try to use auto-fill for grid-template-columns and set max-width for catalog instead of using media-queries

**Review by Anna-Tkachenko — APPROVED** [review]
Good job! 👍

**Pull request closed without merging**